app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
```

Comment:
What does each of these options mean?

Reply:
They are the modules that can be configured from the Devise method in the User model in this case, they are:

database_authenticatable, with this configuration users, will be able to authenticate with a username and password stored in the database.

registerable, so that users can register, update and delete their profiles.

rememberable, to enable the option to remember me on the login

validatable, which validates the User's email and password